package.json

```diff
@@ -1,37 +1,68 @@
 {
-  "name": "load-secrets-action",
-  "version": "1.1.0",
-  "description": "Load Secrets from 1Password",
-  "main": "dist/index.js",
-  "directories": {
-    "test": "tests"
-  },
-  "scripts": {
-    "build": "ncc build src/index.ts"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/1Password/load-secrets-action.git"
-  },
-  "keywords": [
-    "actions",
-    "1password",
-    "load secrets",
-    "connect"
-  ],
-  "author": "1Password",
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/1Password/load-secrets-action/issues"
-  },
-  "homepage": "https://github.com/1Password/load-secrets-action#readme",
-  "dependencies": {
-    "@actions/core": "^1.10.0",
-    "@actions/exec": "^1.1.1"
-  },
-  "devDependencies": {
-    "@types/node": "^18.14.2",
-    "@vercel/ncc": "^0.36.1",
-    "typescript": "^4.9.5"
-  }
+	"name": "load-secrets-action",
+	"version": "1.2.0",
+	"description": "Load Secrets from 1Password",
+	"type": "module",
+	"main": "dist/index.js",
+	"directories": {
+		"test": "tests"
+	},
+	"scripts": {
+		"build": "ncc build ./src/index.ts",
+		"format": "prettier --ignore-path ./config/.prettierignore",
+		"format:check": "npm run format -- --check ./",
+		"format:write": "npm run format -- --write ./",
+		"lint": "eslint ./",
+		"lint:fix": "npm run lint -- --fix",
+		"pre-commit": "lint-staged --config ./config/lint-staged.config.js",
+		"pre-push": "npm run validate",
```

Comment:
~Are these necessary? I thought you wouldn't need to specify these since we're registering Husky hooks in `.husky`? For example, [op-vscode uses a Husky pre-commit hook](https://github.com/1Password/op-vscode/blob/main/package.json#LL27C30-L27C30) and doesn't need to register it in scripts. What's the difference?~

Reply:
Good callout 👍 Resolving this conversation via the https://github.com/1Password/load-secrets-action/pull/34/commits/1eecfadbf27d622b4ecdcac5ec3b274dc9345fa0 commit.